awx/settings/defaults.py

```diff
@@ -1211,3 +1183,21 @@ def IS_TESTING(argv=None):
 
 # Delete temporary directories created to store playbook run-time
 AWX_CLEANUP_PATHS = True
+
+MIDDLEWARE = [  # NOQA
```

Comment:
does this still require NOQA? if you run flake8 on this file without that, does it still give an error?

Reply:
`# NOQA` does not seem to be required.